app/src/main/res/layout/quick_keyboard_button.xml

```diff
@@ -11,17 +11,15 @@
 -->
 <ImageView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
+    android:layout_width="24dp"
+    android:layout_height="24dp"
     android:background="@drawable/keyboard_shortcut_button"
     android:gravity="center_vertical|center_horizontal"
-    android:minWidth="40dp"
-    android:minHeight="48dp"
-    android:paddingStart="12dp"
-    android:paddingLeft="12dp"
+    android:paddingLeft="8dp"
+    android:paddingRight="8dp"
     android:paddingTop="8dp"
-    android:paddingEnd="12dp"
-    android:paddingRight="12dp"
     android:paddingBottom="8dp"
+    android:scaleType="fitCenter"
     android:tint="@color/textActionIconColor"
+    android:src="@drawable/ic_add_white_24dp"
```

Comment:
Default drawable makes it easier to debug the layout. will not show up in practice.

Reply:
tools:src ?

Reply:
Anything, honestly. Just so that there is a default icon I can see in the layout :)

Reply:
then tools: instead android: is exactly whats needed